packages/components/filters/filter-menu/src/filter-menu.tsx

```diff
@@ -0,0 +1,124 @@
+import { type ReactElement } from 'react';
+
+import { Footer } from './footer';
+import { Header } from './header';
+import { TriggerButton } from './trigger-button';
+import * as Popover from '@radix-ui/react-popover';
```

Comment:
I'm really happy with how radix handles the `popover` compared to the `DropdownMenu`.  I think we should use it.

I also think we should consider replacing the `DropdownMenu` internals with the radix `popover` instead of trying to implement complicated bugfixes and a11y updates to it ourselves.